schema_plus_columns.gemspec

```diff
@@ -17,8 +17,9 @@ Gem::Specification.new do |gem|
   gem.test_files    = gem.files.grep(%r{^(test|spec|features)/})
   gem.require_paths = ["lib"]
 
-  gem.add_dependency "activerecord", "~> 4.2"
-  gem.add_dependency "schema_plus_indexes", "~> 0.1"
+  gem.add_dependency "activerecord", ">= 4.2", "< 5.1"
+  gem.add_dependency "schema_plus_core"
+  gem.add_dependency "schema_plus_indexes", "~> 0.2.4"
```

Comment:
This should depend on just "~> 0.2" not "~> 0.2.4" so it'll be upward compatible with schema_plus_indexes 0.3+; and if anybody is using this with AR 5, bundler should ensure that they'll get 0.2.4 anyway, no?